Makefile

```diff
@@ -1667,14 +1674,14 @@ endif
 	   "$(INSTALL_COMPLIBDIR)"
 ifeq "$(INSTALL_SOURCE_ARTIFACTS)" "true"
 	$(INSTALL_DATA) \
-	   utils/*.cmt utils/*.cmti utils/*.mli \
-	   parsing/*.cmt parsing/*.cmti parsing/*.mli \
-	   typing/*.cmt typing/*.cmti typing/*.mli \
-	   file_formats/*.cmt file_formats/*.cmti file_formats/*.mli \
-	   lambda/*.cmt lambda/*.cmti lambda/*.mli \
-	   bytecomp/*.cmt bytecomp/*.cmti bytecomp/*.mli \
-	   driver/*.cmt driver/*.cmti driver/*.mli \
-	   toplevel/*.cmt toplevel/*.cmti toplevel/*.mli \
+	   utils/*.cmt utils/*.cmti $(SRCDIR)/utils/*.mli \
+	   parsing/*.cmt parsing/*.cmti $(SRCDIR)/parsing/*.mli \
+	   typing/*.cmt typing/*.cmti $(SRCDIR)/typing/*.mli \
+	   file_formats/*.cmt file_formats/*.cmti $(SRCDIR)/file_formats/*.mli \
+	   lambda/*.cmt lambda/*.cmti $(SRCDIR)/lambda/*.mli \
+	   bytecomp/*.cmt bytecomp/*.cmti $(SRCDIR)/bytecomp/*.mli \
+	   driver/*.cmt driver/*.cmti $(SRCDIR)/driver/*.mli \
+	   toplevel/*.cmt toplevel/*.cmti $(SRCDIR)/toplevel/*.mli \
```

Comment:
If I understand correctly, the code here assumes that all `.mli` files are source files, but this is not true, looking for `.mli:` in the Makefile suggests that for example utils/domainstate.mli, asmcomp/arch.mli, parsing/parser.mli are generated.